### PR TITLE
fix: Added support for Array schema

### LINKED ISSUE #1: bug: Unable to (de)normalize an array of values
Hi there,

Love the project, always a fan of using decorators to avoid extra schema definitions everywhere!

Ran into an issue though, I can't work out how to (de)normalize to and from an array of values. In normalizr you just wrap the schema in `[]`, however I've dug through your source and cannot find a way to do it.

Are you able to please advise? Thanks!

### PR DESCRIPTION
**Pull request by edcarroll**
Closes #1 

This PR also addresses various other things within the project, like tests & ensuring that the typings are found correctly, as well as removing compiled files from the repo.

### CODE REVIEW

**Comment by edcarroll** [comment]
Are you meaning shrinkwrap vs package-lock? As far as I'm aware package-lock.json is essentially a new version of the shrinkwrap functionality, but is a bit easier to use as it just works with `npm install` without any hassle

**Comment by edcarroll** [comment]
btw, are you meaning to re-approve the PR so much? 😛

**Comment by mchlbrnd** [comment]
Ha, I was looking at the review functionality in GitHub. :P

**Comment by edcarroll** [comment]
Fair enough! lmk if you want me to change anything

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "normalizr-decorators",
-  "version": "1.0.0-alpha.1",
+  "name": "@edcarroll/normalizr-decorators",
```
Comment (mchlbrnd) [inline package.json]: Remove only

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "1.0.0-alpha.5",
   "description": "Schema decorators for normalizr",
   "author": "Michael Wolbert <michael.wolbert@gmail.com>",
```
Comment (mchlbrnd) [inline package.json]: Add yourself here please! :)

**Comment by edcarroll** [comment]
Thank you for sorting that 🎉 currently underwater with a work project, but should be wrapped up on Thursday so will be able to take a look at & have a chat about some of the other issues 😄

**Pull request merged**